README.md

```diff
@@ -1,20 +1,27 @@
 ![CF](http://i.imgur.com/7v5ASc8.png) LAB
 =================================================
+[![Build Status](https://travis-ci.com/401-advanced-javascript-billybunn/lab-09.svg?branch=working)](https://travis-ci.com/401-advanced-javascript-billybunn/lab-09)
+
+## Notes
+* UML diagrams done
+* jsdocs almost complete
+* started swagger, but had trouble completing (and getting it hosted somewhere?)
+* didn't have time to write the tests. also think there's an issue with supergoose
```

Comment:
In your test try importing app instead server const app = require(`${rootDir}/src/app.js`);